packages/collection-model/index.d.ts

```diff
@@ -33,21 +33,24 @@ type CollectionMetadata = {
    */
   sourceName?: string;
   /**
-   * Indicates if a source collection is read only
+   * View source pipeline definition
    */
-  sourceReadonly?: boolean;
+  sourcePipeline?: unknown[];
   /**
-   * View source view namespace (this is the same as metadata namespace if
-   * present)
+   * Instance metadata: whether or not we are connected to the ADF
```

Comment:
This ADF / Atlas Data Lake terms confuse me to this day:
https://www.mongodb.com/blog/post/announcing-atlas-data-federation-data-lake

Here is what? Especially when comment says ADF, and the property isDataLake :)

Reply:
Data Lake was renamed to ADF, so it is called ADF, but even server commands still refer to ADF as data lake (like in build info), so we also still referring to it like that in the app internally.

If we ever would want to change that, this would honesly be a project of it's own, this goes beyond just Compass internals, packages like build-info will need to have a breaking change